internal/v4/archive_test.go

```diff
@@ -283,7 +283,7 @@ func (s *ArchiveSuite) TestConcurrentUploads(c *gc.C) {
 	// by closing errorBodies.
 	try := make(chan struct{})
 	go func(try chan struct{}) {
-		for _ = range try {
+		for range try {
```

Comment:
@mhilton make simplify does that, may be it's 1.4, but Jenkins seems to be 1.2 so let's track when you have the compilation error again.